### PR TITLE
Create infra aws

### PR DESCRIPTION
**Pull request by guillaumem-theodo**

### CODE REVIEW

File: infra/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Set up AWS on your local
+
+### Install AWS CLI
+
+- You need to have AWS CLI installed on your computer : https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html
+
+### Setup IAM user on AWS
+
+- Go to [AWS Single Sign-On](https://theodo.awsapps.com/start/#/) and loggin to your account.
```
Comment (Paulmolin) [inline infra/README.md:9]: log in

File: infra/networking.tf [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  value = aws_subnet.public-subnet-bastion.id
+}
+
+output "private_subnet" {
+  value = aws_subnet.private-subnet.id
+}
+
+output "vpc" {
+  value = aws_vpc.vpc.id
+}
```
Comment (Paulmolin) [inline infra/networking.tf:123]: Pareil

File: infra/README.md (hunk 1 above)
Comment (guillaumem-theodo) [reply infra/README.md]: fixed

File: infra/backend.tf [hunk 3]
```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "s3" {
+    bucket         = "dojo-secu-terraform-states"
+    key            = "infra"
+    region         = "eu-west-2"
+    profile        = "dojo-security"
+  }
+}
```
[the remark this answers is not in the document]
Comment (guillaumem-theodo) [reply infra/backend.tf]: fixed

File: infra/networking.tf (hunk 2 above)
Comment (guillaumem-theodo) [reply infra/networking.tf]: fixed

**Pull request merged**